### PR TITLE
Update README.md to explain how to run the tests

### PR DESCRIPTION
**Pull request by temsa**
(plus cleanup end of line spaces)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,9 +211,16 @@ First, start postgres with:
 $ npm run postgres
 ```
 
-Then in another terminal:
+Then you can, in another terminal, find the running docker, init the DB, then run the tests:
 
 ```
+$ docker ps
```
Comment (cemremengu) [inline README.md]: I dont think we should make the example docker specific so just specifying below should be ok

Then, in another terminal, you can init the test database and run the tests:

```
$ psql -d postgres -U postgres  -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);'
$ npm test
```

Comment (cemremengu) [reply README.md]: @temsa sorry ignore this

**Comment by cemremengu** [comment]
@mcollina your thoughts?

File: README.md [hunk 2]
```diff
@@ -211,9 +211,16 @@ First, start postgres with:
...
+CONTAINER ID        IMAGE                 COMMAND                  CREATED             STATUS              PORTS                    NAMES
+28341f85c4cd        postgres:9.6-alpine   "docker-entrypoint.s…"   3 minutes ago       Up 3 minutes        0.0.0.0:5432->5432/tcp   jovial_shockley
+
+$ docker exec -ti jovial_shockley /usr/local/bin/psql -d postgres -U postgres  -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);'
+CREATE TABLE
```
Comment (cemremengu) [inline README.md]: @temsa just noticed, is this left over by any chance?

Comment (temsa) [reply README.md]: @cemremengu it is not, is is just the output from the `psql` command, like the `docker ps` one above it :)

Comment (cemremengu) [reply README.md]: Thanks, I need more coffee :)

**Pull request merged**